package/expo-package/src/handlers/Video.tsx

```diff
@@ -1,11 +1,11 @@
 import { VideoComponent } from '../optionalDependencies/Video';
 
 export const Video = VideoComponent
-  ? ({ onPlaybackStatusUpdate, paused, style, uri, videoRef }) => (
+  ? ({ onPlaybackStatusUpdate, paused, resizeMode, style, uri, videoRef }) => (
       <VideoComponent
         onPlaybackStatusUpdate={onPlaybackStatusUpdate}
         ref={videoRef}
-        resizeMode='contain'
+        resizeMode={resizeMode}
```

Comment:
What does it default to?

Reply:
`contain`

Check https://github.com/GetStream/stream-chat-react-native/blob/develop/package/src/components/ImageGallery/components/AnimatedGalleryVideo.tsx#L198